File: 009.palindrome-number/009.palindrome-number.java
public boolean isPalindrome(int x) {
        if (x < 0) {
            return false;
        }
        int n = x;
        int tmp  = 0;
        while (x != 0) {
            tmp = tmp * 10 + x % 10;
            x = x/10;
        }
        return n == tmp ? true : false;
    }

File: 009.palindrome-number/009.palindrome-number.py
class Solution:
    def isPalindrome(self,x):
        """
        :type x: int
        :rtype: bool
        """
        if(x<0):
            return False
        n = x;
        temp = 0;
        while x!=0:
            temp = temp * 10 + x%10;
            x=x//10;        
        return n==temp
